src/tests/ftest/ior/hard_rebuild.yaml

```
@@ -53,6 +53,7 @@ ior:
       # - [EC_Object_Class, Minimum number of servers]
       - ["EC_2P2GX", 6]
       - ["EC_4P2GX", 8]
+      - ["EC_4P3GX", 9]
```

Again, comment above in this config is wrong
```suggestion
      - ["EC_4P3GX", 12]
```